api.go

```diff
@@ -726,6 +738,16 @@ func (api *API) ImportValue(_ context.Context, req *ImportValueRequest) error {
 	return errors.Wrap(err, "importing")
 }
 
+func importExistenceColumns(index *Index, columnIDs []uint64) error {
```

Comment:
is the Index expected to be locked when this helper is called? we should document that if so. If not, how are we able to call an index.unprotected* method?

Reply:
I believe this is addressed now with the `RLock()` in the `index.existenceField()` method.

Also, out of curiosity, I tried deleting both the import field and the `exists` field DURING an import. Deleting the actual field during an import into that field resulted in the import process aborting with `err=field not found`. Deleting the `exists` field during an import continued with the import and resulted in no error (i.e. the `exists` field was safely removed and the import completed successfully).

Reply:
that all seems reasonable - good idea on those tests. I'll finish reviewing this now.